release.yaml

```diff
@@ -24,6 +24,7 @@
 # release requirements, don't forget to register check function in a file requirements.go
 requirements:
   "k8s": "1.25.0" # modules/040-control-plane-manager/requirements/check.go
+  "k8sMinimumNotAllowedVersion": "1.27.0" # modules/340-monitoring-kubernetes/requirements/check.go
```

Comment:
It must be in the next release

Reply:
I'm basing this on the fact that version 1.27 is already installed as the default version of kubernates.
https://github.com/deckhouse/deckhouse/blob/main/dhctl/pkg/config/base.go#L39